contracts/test/forkMainnet/AMMWrapper/SetAndCloseAllowance.t.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import "contracts-test/forkMainnet/AMMWrapper/Setup.t.sol";
+
+contract TestAMMWrapperSetAndCloseAllowance is TestAMMWrapper {
+    function testCannotSetAndCloseAllowanceByNotOperator() public {
```

Comment:
Maybe this test could be split into two: `testCannotSetByNotOperator`, `testCannotCloseByNotOperator`

Reply:
And in my personal preference, I would call this test contract as `TestAMMWrapperAllowance`, since this suite contains multiple actions related to allowance